### ISSUE TITLE
Broken build process

### ISSUE THREAD

**Issue by AlessandroSangiuliano** [etoile/XMPPKit#1]
XMPPKit can't build on GNUstep because of the absence of AddressBook.

**Comment by qmathe** [comment]
The XMPPKit INSTALL file should probably include a note explaining Addresses framework is required and provide a download link.

In the default Etoile build process, XMPPKit is turned off though. Framework/GNUmakefile does the following: export xmppkit ?= no

**Comment by davidchisnall** [comment]
I think that we should probably deprecate XMPPKit a bit more forcefully.  XMPPFramework has made a huge amount of progress and is now much better than XMPPKit.  Focusing effort on ensuring that XMPPFramework works well on GNUstep is probably more valuable than continuing to maintain XMPPKit.

**Comment by qmathe** [comment]
I agree about moving to XMPPFramework, CoreObject uses it currently and it works really well.

**Comment by AlessandroSangiuliano** [comment]
Ok, after my OS X laptop hard disk completely died, I can answer here. 
Some years ago (1 or 2) I was testing some piece of code of CO with Eric, I don't remember what but I remember that we started to talk about switching from XMPPKit to XMPPFramewrok, the main reason was that CO is and was using XMPPFramework demonstrating that framework, at the time, was quite stable working well; also, it had some features I never was able to retail the right amount of time to work on, like MUC, implemented. So, I think we should consider to switch to XMPPFramework; however months ago I asked to them if they have some plan to port they framework to GNUstep. They said that would be interesting XMPPFramework on GNUstep, but they will not port it on GNUstep and said me that we should do the port.

Here on Github my question: https://github.com/robbiehanson/XMPPFramework/issues/551

**Comment by davidchisnall** [comment]
I think that's a fair response.  Most of the 'port' should involve fixing GNUstep bugs so that it works out of the box.

**Comment by AlessandroSangiuliano** [comment]
I'd like to help doing this contributing directly to GNUstep too, in the free time. However I'm a bit discouraged about the actual GNUstep situation, to me its future isn't clear and despite there's someone that is pushing on the ML to take GNUstep, a step (sorry for this word pun) looking to the future, there are a lot of people in that project, very happy of the actual GNUstep situation, seeing it like their personal toy (as was the the HURD about 12 years ago, then near the death, they started to open their mind about 2 - 3 years ago, and now they are able to attract new developers and contributors, reached the 0.7 release and almost completed the port of the NetBSD rump kernel on the HURD). Instead on GNUstep, each time someone propose something of new, like a stupid theme or just switching to a new host site like GitHub (full of people) or starting to consider Wayland, is ignored, lynched or just refused, and ever by the same people that are living in the past. That's a bit ( a lot ) frustrating, so is quite difficult to me asking for commit rights in the GNUstep repo.

I really hope someone will fork the GNUstep project, entirely.
